iphone/Classes/TiAppiOSUserActivityProxy.m

```diff
@@ -425,6 +438,73 @@ - (void)resignCurrent:(id)unused
   }
   [_userActivity resignCurrent];
 }
+
+#if IS_XCODE_10
+
+- (NSNumber *)eligibleForPrediction
+{
+  if (![TiUtils isIOSVersionLower:@"12.0"]) {
+    return NUMBOOL(NO);
+  }
+
+  return NUMBOOL(_userActivity.isEligibleForPrediction);
+}
+
+- (void)setEligibleForPrediction:(NSNumber *)value
+{
+  ENSURE_TYPE(value, NSNumber);
+  ENSURE_UI_THREAD(setEligibleForSearch, value);
+  if (![TiUtils isIOSVersionLower:@"12.0"]) {
+    return;
+  }
+  [_userActivity setEligibleForPrediction:[TiUtils boolValue:value]];
+}
+
+- (NSString *)persistentIdentifier
+{
+  if (![TiUtils isIOSVersionLower:@"12.0"]) {
+    return nil;
+  }
+
+  return _userActivity.persistentIdentifier;
+}
+
+- (void)setPersistentIdentifier:(NSString *)value
+{
+  ENSURE_TYPE(value, NSString);
+  if ([TiUtils isIOSVersionLower:@"12.0"]) {
+    return;
+  }
+  [_userActivity setPersistentIdentifier:[TiUtils stringValue:value]];
+}
+
+- (void)deleteSavedUserActivitiesForPersistentIdentifiers:(id)persistentIdentifiers
+{
+  ENSURE_ARRAY(persistentIdentifiers);
```

Comment:
Did you actually test all the new API's so far? Since this is a method, you need to unbox the `persistentIdentifiers` attribute with `ENSURE_SINGLE_ARG(persistentIdentifiers, NSArray)`.